conf/nginx/st2.conf

```diff
@@ -135,6 +137,8 @@ server {
     root      /opt/stackstorm/static/webui/;
     index     index.html;
 
+    proxy_set_header Range "";
```

Comment:
How about the other locations above (i.e. /auth/, /api/, etc.)?

Reply:
Is this more of a hypothetical / possible vulnerability or the code is actually vulnerable to this issue?

Because afaik, st2api doesn't handle ``Range`` header anywhere (so it should just be ignored). Or does it apply to nginx server static files?